src/initialize-test-window.coffee

```diff
@@ -57,6 +57,12 @@ module.exports = ({blobStore}) ->
 
     document.title = "Spec Suite"
 
+    # Avoid throttling of test window by playing silence
+    context = new AudioContext()
+    source = context.createBufferSource()
+    source.connect(context.destination)
+    source.start(0)
+
     testRunner = require(testRunnerPath)
```

Comment:
@nathansobo `disable-background-timer-throttling` only disables throttling for timers, for `requestAnimationFrame` it doesn't work. I tried lots of ways but still can not find the correct way to make `requestAnimationFrame` work for background pages, we probably need help from Chrome developers.